card/lib/card/format/render.rb

```diff
@@ -122,12 +122,14 @@ def view_setting setting_name, view
       end
 
       def stub_render cached_content
-        result = expand_stubs cached_content do |stub_hash|
-          prepare_stub_nest(stub_hash) do |stub_card, view_opts|
-            nest stub_card, view_opts, stub_hash[:format_opts]
-          end
+        stub_debugging do
+          expand_stubs cached_content
         end
-        if result =~ /stub/
+      end
+
+      def stub_debugging
+        result = yield
+        if Rails.env.development? && result =~ /stub/
```

Comment:
if we keep this, we should add a config option rather than always debugging in development.